release.py

```diff
@@ -325,6 +326,23 @@ def sanity_check_sdformat_versions(package, version):
 
     print_success("sdformat version in proper sdformat package")
 
+def sanity_cmake_version(version):
```

Comment:
```suggestion
def sanity_check_cmake_version(version):
```
nit (I know the method names aren't consistent, but I think it's clearer to start with `sanity_check_`)

Reply:
Done in https://github.com/gazebo-tooling/release-tools/pull/1224/commits/0a7b4307325d2e35765b6da0801859535c3045d5